tee attributes.md

```diff
@@ -0,0 +1,16 @@
+## TEE Attributes
+
+Required properties to be called a “TEE”:
+* ___Data integrity___ - data values stored or being operated upon within the bounds of the TEE cannot be modified by an actor operating outside the TCB.
+* ___Data confidentiality___ - data values stored or being operated upon within the bounds of the TEE cannot be determined by an actor operating outside the TCB.
+* ___Code integrity___ - software codes executed inside the bounds of the TEE cannot be modified by an actor operating outside the TCB.
+
+TEE's may have other properties:
+* _Code confidentiality_ - software codes executed inside the bounds of the TEE cannot be modified by an actor operating outside the TCB.
+* _Programmability_ - the extent to which the software codes operated upon inside the bounds of the TEE can be provided by an identified actor.
+* _Attestability_ - the mechanism for demonstrating the contents of a TEE is infact secured by a TEE.
```

Comment:
```suggestion
* _Attestability_ - the mechanism for demonstrating the contents of a TEE is in fact secured by a TEE.
```

Reply:
I'd also compare the text currently in the whitepaper, which might have a better definition